lesson_00/NileJackson/README.md

```diff
@@ -0,0 +1,49 @@
+
+## Likes
+
+I enjoy cooking because I'm an eight year vegan who misses the dishes I grew up on, namely jamaican food; I spend a lot of down time in the kitchen trying to "veganize" things. Other than food, I love to read, I hated books as a kid but four/five years ago I moved to Iowa to play(school) football at a small private college called William Penn, I wasn't a fan of Iowa or the school so I began using the library as a place to escape. I also like my two dogs, Pico(a pitbull/great dane mix) & LuLu(a small mut of some sort).
+## Dislikes
+I'm not a fan of the following; 
+* artichokes 
+* english breakfast tea 
+* the new york subway system 
+* excessively cold winters. 
+
+I'm not a fan of the following; artichokes, english breakfast tea, the new york subway system, and excessively cold winters.
+
+## Hobbies
+
+In my free time I enjoy
+
+ * Various forms of exercise (force of habit from a life of sport)
+ * reading ( right now I'm reading a book called cosmos and psyche by richard tarnas)
+ * anime/manga (fav anime: berserk, fav manga: sakamoto days)
+ * lounging/napping with my dogs pico and lulu
+ * My working/learning style: Myers briggs says I'm an INTP. It usually takes me some time to open up to new people; peers/coworkers, but after that I'm pretty much an open book. I'm a visual learner who values repition. Professionally, I'm not sure what my style is, but I'm working on that.
+
+My circadian rhythm: I'm a night owl, but I'd like to change that, as most of my nights tend to spill into days. This is also a work in progress.
+
+Why I value programming: I'm sure this will change over time, but for now I'll say I value programming because it's empowering; I went to middle/high school for fine art because I loved making "something from nothing", and the idea of making a program stirs up that same feeling.
+
+Moving forward: I look forward to learning github, git, and docker because I don't enjoy feeling confused. I also hope I can make myself an asset to the other members of my cohort because that would mean that I have a clue as to what's going on. My main goal here at code differently is to gain programming skills, get better at problem solving, build relationships, and get a job so I can get paid to learn.
+
+Cool Pictures
```

Comment:
Look like a duplicate line of text, please remove (given that you have the correct header on the next line below).